doc/accounts.md

```diff
@@ -0,0 +1,177 @@
+ - The "spend up to a certain amount automatically" model: A web user has a
+   browser extension like [Alby](https://getalby.com/) installed and wants to
+   allow that extension to pay invoices for paywalls automatically up to a
+   certain amount per month. That amount could be enforced by the account so the
+   browser extension doesn't have to keep track of its spending actions. And an
+   account can be shared between extensions installed in different browsers.
+ - The "allowance" model: A parent wants to give their child their allowance in
+   Lightning satoshis. They create an account over the allowance amount and top
+   up the account each week/month.
+
+## HOWTO
+
+This section describes how an account can be created and used.
+
+### Create the account
+
+The first thing that needs to be done is to create the account with its initial
+balance (and an optional expiry). This **needs to be done by the node
+operator**, meaning access to the `lit.macaroon` is required.
+
+Example:
+```shell
+$ litcli accounts create 50000 --save_to /tmp/accounts.macaroon
+
+{
+        "account": {
+                "id": "d64dbc31b28edf66",
+                "initial_balance": "50000",
+                "current_balance": "50000",
+                "last_update": "1652353332",
+                "expiration_date": "0"
+        },
+        "macaroon": "020103........."
+}
+Account macaroon saved to /tmp/accounts.macaroon
+```
+
+This created a new account (ID `d64dbc31b28edf66`) with an initial balance of
+50k satoshis and no expiration. A new macaroon was baked that contains the
+correct permissions and is locked to that account. The macaroon file was stored
+under `/tmp/accounts.macaroon` in this example.
+
+### Use the macaroon
+
+This step is done by the user/app that should be given the restricted access. An
+example could be to create a QR code with a tool like
+[`lndconnect`](https://github.com/LN-Zap/lndconnect) that can be scanned by
+mobile apps to connect to the node. Or some browser extensions require the user
+to upload the macaroon to the browser.
+
+**It's absolutely crucial to use the macaroon generated in the previous step
+here** to make sure the restrictions are applied.
+
+The permissions and restrictions of a macaroon can always be inspected by:
+```shell
+$ lncli printmacaroon --macaroon_file /tmp/accounts.macaroon
+
+{
+        "version": 2,
+        "location": "lnd",
+        "root_key_id": "0",
+        "permissions": [
+                "info:read",
+                "invoices:read",
+                "invoices:write",
+                "offchain:read",
+                "offchain:write",
+                "onchain:read"
+        ],
+        "caveats": [
+                "lnd-custom account d64dbc31b28edf66"
+        ]
+}
+```
+
+The important part is the `lnd-custom account ...` part in the `caveats`
+section.
+
+Example of using `lncli` to check the account balance:
+```shell
+$ lncli --macaroonpath=/tmp/accounts.macaroon channelbalance
```

Comment:
maybe worth noting that this is only in integrated mode (right?)

Reply:
No, this should also work in remote mode. But then of course the server and TLS cert need to be adjusted. Added that as a note.